Sources/MockoloFramework/Templates/ClosureTemplate.swift

```diff
@@ -32,6 +32,9 @@ func applyClosureTemplate(name: String,
             if argType.typeName.hasPrefix(String.autoclosure) {
                 return argName + "()"
             }
+            if argName.isSwiftKeyword {
```

Comment:
add handling for an argument following a keyword (see my comment on test below for an example)

Reply:
Will add these tests today